src/languages/gams.js

```diff
@@ -0,0 +1,30 @@
+module.exports = function (hljs){
+
+    var keywords = 'abort acronym acronyms alias all and assign binary card diag display else1 eps eq equation equations file ' +
+        'files for1 free ge gt if inf integer le loop lt maximizing minimizing model models na ne negative no not option   options or ord ' +
+        'parameter parameters positive prod putpage puttl repeat sameas scalar scalars semicont semiint set1 sets smax smin solve sos1 sos2 ' +
+        'sum system table then until using variable variables while1 xor yes';
+
+    keywords += keywords.toUpperCase();
```

Comment:
It's easier to just set `case_insensitive: true` instead of duplicating keywords.